messages/context.json

```diff
@@ -0,0 +1,12 @@
+{
+  "shippingEstimate-bd": "Shipping estimates in business days",
+  "shippingEstimate-d": "Shipping estimates in days",
+  "shippingEstimate-h": "Shipping estimates in hours",
+  "shippingEstimate-m": "Shipping estimates in minutes",
+  "shippingEstimate-scheduled": "Shipping delivery window",
+  "shippingEstimate-scheduled-no-dates": "Scheduled",
+  "shippingEstimatePickup-bd": "Store pickup estimates in business days",
+  "shippingEstimatePickup-d": "Store pickup estimates in days",
+  "shippingEstimatePickup-h": "Store pickup estimates in hours",
+  "shippingEstimatePickup-m": "Store pickup estimates in minutes"
```

Comment:
I think all of those description might be "estimate".

Like: "Shipping estimate in hours", "Store pickup estimate in days", etc.

Reply:
Yep! Makes sense hahahaha
Updated it :)